<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>983</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1420</width>
    <height>920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sennalabs</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_window">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>1360</width>
      <height>850</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="register_button">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>900</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/add_button.png</normaloff>../assets/add_button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="history_button">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>900</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>History</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/history.png</normaloff>../assets/history.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="checkin_label">
    <property name="geometry">
     <rect>
      <x>1503</x>
      <y>120</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>630</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="member_button">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>900</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Member</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/user-icon.webp</normaloff>../assets/user-icon.webp</iconset>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time_lcd">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>10</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>280</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/Sennalabs.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="check_widget" native="true">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>230</y>
      <width>341</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
